strawberry/schema/schema.py

```diff
@@ -283,7 +285,7 @@ def _create_execution_context(
             provided_operation_name=operation_name,
         )
 
-    @lru_cache
+    @lru_cache  # noqa: B019
```

Comment:
https://docs.astral.sh/ruff/rules/cached-instance-method/

very sceptical about enabling the rule here, as many instances here have potentially infinite lifetime and we _need_ caching on them. My concern is it may discourage PR submitters from caching where it makes sense

Reply:
Good point! Going to disable this one globally